### PR TITLE
Addressing comments from #703

### PR DESCRIPTION
**Pull request by AnmAtAnm**
This change is [Reviewable](https://reviewable.io/reviews/google/blockly/719)

### CODE REVIEW

File: core/workspace_svg.js [hunk 1]
```diff
@@ -999,7 +999,7 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
         deleteNext();
       } else {
         Blockly.confirm(Blockly.Msg.DELETE_ALL_BLOCKS.
-            replace('%1',String(deleteList.length)),
+            replace('%1', String(deleteList.length)),
```
Comment (NeilFraser) [inline core/workspace_svg.js]: String() isn't actually needed.  Because JavaScript.

File: demos/custom-dialogs/custom-dialog.js [hunk 2]
```diff
@@ -74,90 +81,91 @@ CustomDialog.hide = function() {
...
+    CustomDialog.dialogDiv_ = dialogDiv;
+  }
+  backdropDiv.style.display = 'block';
+  dialogDiv.style.display = 'block';
+
+  dialogDiv.innerHTML =
+      '<header class="customDialogTitle"></header>' +
+      '<p class="customDialogMessage"></p>' +
+      (options.showInput? '<div><input id="customDialogInput"></div>' : '') +
+      '<div class="customDialogButtons">' +
```
Comment (NeilFraser) [inline demos/custom-dialogs/custom-dialog.js]: This class is never used.

File: demos/custom-dialogs/custom-dialog.js [hunk 3]
```diff
@@ -74,90 +81,91 @@ CustomDialog.hide = function() {
...
+        return false;
+      }
+    };
+  } else {
+    var okay = document.getElementById('customDialogOkay');
+    okay && okay.focus();
+  }
+
+  if (options.showOkay) {
+    document.getElementById('customDialogOkay').onclick = onOkay;
```
Comment (NeilFraser) [inline demos/custom-dialogs/custom-dialog.js]: It doesn't matter in this case, but onclick is not the best way to set an event as it only allows a single event per element.  The recommended method is:
document.getElementById('customDialogOkay').addEventListener('click', onOkay);

File: demos/custom-dialogs/custom-dialog.js [hunk 4]
```diff
@@ -74,90 +81,91 @@ CustomDialog.hide = function() {
...
+    event && event.stopPropagation();
+  };
+
+  var dialogInput = document.getElementById('customDialogInput');
+  CustomDialog.inputField = dialogInput;
+  if (dialogInput) {
+    dialogInput.focus();
+
+    dialogInput.onkeyup = function(event) {
+      if (event.keyCode == 13) {
```
Comment (NeilFraser) [inline demos/custom-dialogs/custom-dialog.js]: Space is usually also assigned to Ok (at least in Windows).

File: demos/custom-dialogs/custom-dialog.js [hunk 5]
```diff
@@ -74,90 +81,91 @@ CustomDialog.hide = function() {
...
+    CustomDialog.backdropDiv_ = backdropDiv;
+    CustomDialog.dialogDiv_ = dialogDiv;
+  }
+  backdropDiv.style.display = 'block';
+  dialogDiv.style.display = 'block';
+
+  dialogDiv.innerHTML =
+      '<header class="customDialogTitle"></header>' +
+      '<p class="customDialogMessage"></p>' +
+      (options.showInput? '<div><input id="customDialogInput"></div>' : '') +
```
Comment (NeilFraser) [inline demos/custom-dialogs/custom-dialog.js]: Missing space before question mark.

File: demos/custom-dialogs/custom-dialog.js (hunk 4 above)
Comment (AnmAtAnm) [inline demos/custom-dialogs/custom-dialog.js]: This is if the button has focus, in which the default button behavior already does this.

File: demos/custom-dialogs/custom-dialog.js (hunk 2 above)
Comment (AnmAtAnm) [inline demos/custom-dialogs/custom-dialog.js]: Available for people to set styling. For example, right alignment.

**Comment by AnmAtAnm** [comment]
Review status: 0 of 2 files reviewed at latest revision, 5 unresolved discussions.

---

*[core/workspace_svg.js, line 1002 at r1](https://reviewable.io:443/reviews/google/blockly/719#-KVC891BFO4s_7b8_MTl-r1-1002:-KVC8Euh8yuJ5rB8cxB0:b-896fix) ([raw file](https://github.com/google/blockly/blob/04593a7424180193ba46bf081e0f4a113b630d50/core/workspace_svg.js#L1002)):*
<details><summary><i>Previously, NeilFraser (Neil Fraser) wrote…</i></summary>
> String() isn't actually needed.  Because JavaScript.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/719)*

**Comment by AnmAtAnm** [comment]
Review status: 0 of 2 files reviewed at latest revision, 5 unresolved discussions.

---

*[demos/custom-dialogs/custom-dialog.js, line 164 at r1](https://reviewable.io:443/reviews/google/blockly/719#-KVC891BFO4s_7b8_MTm-r1-164:-KVC8I6xdnM10h3urupK:bt2vr1b) ([raw file](https://github.com/google/blockly/blob/04593a7424180193ba46bf081e0f4a113b630d50/demos/custom-dialogs/custom-dialog.js#L164)):*
<details><summary><i>Previously, NeilFraser (Neil Fraser) wrote…</i></summary>
> It doesn't matter in this case, but onclick is not the best way to set an event as it only allows a single event per element.  The recommended method is:
> 
> document.getElementById('customDialogOkay').addEventListener('click', onOkay);
</details>
Fixed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/719)*

**Pull request merged**